tutorials/webide-customize/webide-customize.md

```diff
@@ -1,24 +1,27 @@
 ---
 title: Explore and Customize Your SAP Web IDE Full-Stack
-description: Get to know your new development environment by setting some of the preferences.
+description: Get to know your development environment by setting some of the preferences.
 author_name: Shira Manas
 author_profile: https://github.com/shiramanas
 auto_validation: true
-primary_tag: products>sap-web-ide
-tags: [  tutorial>beginner, topic>cloud, products>sap-cloud-platform, products>sap-web-ide, tutorial>license ]
+primary_tag: software-product>sap-web-ide
+tags: [  tutorial>beginner, topic>cloud, software-product>sap-business-technology-platform, software-product>sap-web-ide, tutorial>license ]
 time: 10
 ---
 
 ## Prerequisites  
- - SAP Web IDE Full-Stack is enabled
+- SAP Web IDE Full-Stack is enabled
 
 ## Details
 ### You will learn  
-  - How to change your settings
-  - How to enable new extensions
-  - How to create a new workspace
+- How to change your settings
+- How to enable new extensions
+- How to create a new workspace
 
- In this tutorial, you'll set your theme preferences, connect to your trial Cloud Foundry endpoint on SAP Cloud Platform, and enable some extensions.
+
+In this tutorial, you'll set your theme preferences, connect to your trial Cloud Foundry endpoint on SAP Cloud Platform, and enable some extensions.
+
+We would like to invite you to switch to SAP Business Application Studio. It supports the SAP Web IDE Full-Stack scenarios, including SAP HANA development, providing an improved development experience. New features are added regularly to SAP Business Application Studio to ensure it meets the needs of full stack and cloud application developers.
```

Comment:
Is this tutorial about Web IDE or BAS? As far as I know Web IDE is discontinued. Maybe it would be good to make a BAS version of this tutorial and retire this Web IDE version.

Reply:
SAP Web IDE is not discontinued. We may have more SAP Web IDE users than BAS from what I've heard. Raz asked that we leave this tutorial, but also add a paragraph encouraging users to move to BAS. Since BAS is best for CF, Raz asked me to remove that step here.